### PR TITLE
Add local.py, django-storages for s3 images and statics

### PR DESCRIPTION
**Pull request by jnayak1**
- [x] Add local.py settings configuration
- [x] Add django-storages for s3 images and statics
- [x] Save images locally if developing locally
- [x] Add Author headshot
- [x] Fix pep8 issues
- [x] Add invoke command so that we can test for flake8 locally before pushing

Note:
Statics were uploaded to s3 because it was super easy. We can switch over if necessary any time. Looking into it a bit, I don't think there's a big performance difference. The html page gets sent to the client and then the client requests the static files the page specifies. I don't think there would be a huge difference at least for now whether we serve those from s3 or our ec2 server.

### CODE REVIEW

**Comment by leodomingo** [comment]
Some PEP8 issues here, Travis has a full readout

File: mysite/unicorn/models.py [hunk 1]
```diff
@@ -31,6 +32,10 @@ def __str__(self):
     class Meta:
         ordering = ('text',)
```
Comment (leodomingo) [inline mysite/unicorn/models.py]: Could we switch this over to last_name?

File: mysite/unicorn/models.py [hunk 2]
```diff
@@ -6,6 +6,7 @@
 
 
 class Author(models.Model):
```
Comment (leodomingo) [inline mysite/unicorn/models.py]: Also could you add an image field here for author headshot

File: mysite/unicorn/models.py (hunk 1 above)
Comment (jnayak1) [inline mysite/unicorn/models.py]: Yeah, thats what should be

File: mysite/unicorn/models.py (hunk 2 above)
Comment (jnayak1) [inline mysite/unicorn/models.py]: Will do

**Pull request merged**